src/components/common/program-section-templates/single-image-right/ViewSingleImageRight.module.scss

```
@@ -56,7 +56,7 @@ $img-height: 600px;
 
 .description {
     @include type.body-1-medium;
-    word-wrap: break-word;
+    overflow-wrap: break-word;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)